docs: improve Introduction docs (#1310)
lucide-icons/lucide: docs/index.md

---

# Introduction

> Lucide is an open-source icon library for displaying icons and symbols in both digital and non-digital projects. It offers a collection of 1000+ vector (SVG) files. To simplify the integration of these icons into projects, Lucide provides several official packages.

Reviewer: ```suggestion
Lucide is an open-source icon library to be used in both digital and non-digital projects. It offers a collection of 1000+ icons and symbols in the form of vector images (SVG files). To simplify the integration of these icons into projects, Lucide provides several official packages.
```

Lucide encompasses icons with various variants and states, allowing designers and developers to select the most suitable icon for their needs. If a desired icon is not available, users are encouraged to submit a design request. The Lucide community contributors are dedicated to creating new icons and fulfilling such requests.

> By expanding the range of available icons, Lucide provides users with a broader selection for their projects. Furthermore, as new applications with specific features emerge, Lucide aims to offer a comprehensive icon set to meet diverse project requirements.

Reviewer: ```suggestion
By expanding the range of available icons, Lucide provides users with a broader selection for their projects. Furthermore, as new applications with specific features emerge, Lucide aims to be a comprehensive icon set to meet diverse project requirements.
```

> When designing new icons, the community adheres to a set of design rules to ensure consistency and usability. While creativity is encouraged, maintaining recognizable design conventions is paramount.

Reviewer: ```suggestion
When designing new icons, the community adheres to a strict set of design rules to ensure consistency and legibility on all sizes. While creativity is encouraged, easily recognizable design is of greater importance to maintain our [design conventions](https://lucide.dev/docs/icon-design-guide).
```
